django/db/models/functions/math.py

```diff
@@ -0,0 +1,158 @@
+import math
+
+from django.db.models import FloatField, Func, Transform
+
+
+class Abs(Transform):
+    function = 'ABS'
+    lookup_name = 'abs'
+
+
+class ACos(Transform):
+    function = 'ACOS'
+    lookup_name = 'acos'
+    output_field = FloatField()
+
+
+class ASin(Transform):
+    function = 'ASIN'
+    lookup_name = 'asin'
+    output_field = FloatField()
+
+
+class ATan(Transform):
+    function = 'ATAN'
+    lookup_name = 'atan'
+    output_field = FloatField()
+
+
+class ATan2(Func):
+    function = 'ATAN2'
+    arity = 2
+    output_field = FloatField()
+
+
+class Ceil(Transform):
+    function = 'CEILING'
+    lookup_name = 'ceil'
+
+    def as_oracle(self, compiler, connection):
+        return super().as_sql(compiler, connection, function='CEIL')
+
+
+class Cos(Transform):
+    function = 'COS'
+    lookup_name = 'cos'
+    output_field = FloatField()
+
+
+class Cot(Transform):
+    function = 'COT'
+    lookup_name = 'cot'
+    output_field = FloatField()
+
+    def as_oracle(self, compiler, connection):
+        return super().as_sql(compiler, connection, template='1 / TAN(%(expressions)s)')
+
+
+class Degrees(Transform):
+    function = 'DEGREES'
+    lookup_name = 'degrees'
+    output_field = FloatField()
+
+    def get_degrees(self):
+        degrees = math.degrees(self.source_expressions[0])
+        return float(degrees)
+
+    def as_oracle(self, compiler, connection):
+        expressions = [self.get_degrees()]
+        clone = self.copy()
+        clone.set_source_expressions(expressions)
+        return super(Degrees, clone).as_sql(compiler, connection)
```

Comment:
This is clunky - the extra function, assignments, unnecessary casting to `float()`.

Regardless, this will not work. What if the source expression is `Value(5.5)` or some other complex expression?

I think this is what you are looking for:

```python
def as_oracle(self, compiler, connection):
    return super().as_sql(compiler, connection, template='((%%(expressions)s) * %s / 180)' % math.pi)
```